testing/containers/ceph/Dockerfile

```diff
@@ -0,0 +1,26 @@
+
+ARG CEPH_VERSION
+FROM ceph/daemon-base:latest-${CEPH_VERSION:-nautilus}
+
+ENV CEPH_VERSION=${CEPH_VERSION:-nautilus}
+
+RUN true && \
+    yum install -y git wget curl libcephfs-devel librados-devel librbd-devel /usr/bin/cc /usr/bin/c++ make && \
+    true
+
+ENV GOTAR=go1.12.17.linux-amd64.tar.gz
```

Comment:
For some other tests, I'm using this:

```
rpm --import https://mirror.go-repo.io/centos/RPM-GPG-KEY-GO-REPO
curl -s https://mirror.go-repo.io/centos/go-repo.repo > /etc/yum.repos.d/go-repo.repo
yum -y install golang-1.12
```

In case additional dependencies are needed, using yum/rpm should install those too.

Reply:
I prefer the current approach because it is closer to what we have in the current container builds and I don't really know much about that source. Plus, when I look at the main page it only seems to "know" about go 1.11 and 1.12, which strikes me as a bit stale. I'm not dead set against using a repo/copr for go but I would prefer not to change it in this PR for now.